src/DssSpell.sol

```diff
@@ -18,16 +18,52 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import "dss-interfaces/dss/GemJoinAbstract.sol";
+import "dss-interfaces/ERC/GemAbstract.sol";
 
-import { DssInstance, MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3MAavePoolConfig, D3MAaveBufferPlanConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MCoreInstance } from "src/dependencies/dss-direct-deposit/D3MCoreInstance.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
+interface Initializable {
+    function init(bytes32) external;
+}
+
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface RwaUrnLike {
+    function lock(uint256) external;
+    function vat() external view returns(address);
+    function jug() external view returns(address);
+    function gemJoin() external view returns(address);
+    function daiJoin() external view returns(address);
+    function outputConduit() external view returns(address);
+    function hope(address) external;
+}
+
+interface RwaJarLike {
+    function chainlog() external view returns(address);
+    function dai() external view returns(address);
+    function daiJoin() external view returns(address);
+}
+
+interface RwaOutputConduitLike {
+    function dai() external view returns(address);
+    function gem() external view returns(address);
+    function psm() external view returns(address);
+    function file(bytes32 what, address data) external;
+    function hope(address) external;
+    function mate(address) external;
+    function kiss(address) external;
+}
 
-interface PoolConfiguratorLike {
-    function setReserveFreeze(address asset, bool freeze) external;
-    function setReserveInterestRateStrategyAddress(address asset, address newRateStrategyAddress) external;
-    function setReserveFactor(address asset, uint256 newReserveFactor) external;
+interface RwaInputConduitLike {
+    function dai() external view returns(address);
+    function gem() external view returns(address);
+    function psm() external view returns(address);
+    function to() external view returns(address);
+    function mate(address usr) external;
+    function file(bytes32 what, address data) external;
```

Comment:
Again doesn't match dss-interfaces but does match on-chain. Good job if that's the source you used, but this is a note for me to fix the interfaces...